reports/final/report.tex

```diff
@@ -1675,19 +1719,20 @@ \section{Overall Team Effort}
 % NOTE: I think we will need to add some further reflection here about how the
 % project went overall, it is in the rubric
 
+% if you would like me to revise this effort table, please let me know
```

Comment:
You should not be editing the effort table at this time.

Reply:
Let Ethan do it at the end.

Reply:
a88c1ae5a9bae3099c6a182a36564f77b4af1bc1